README.md

```diff
@@ -3,4 +3,61 @@
 
 # Welcome to Camunda Issues 
 
-In this repository you can get an overview of key features we are currently working on (or planning to work on in the near future). Please don't hesitate to comment any questions or feedback you have directly on the respective issues.
+❇️ View the official [Camunda public product roadmap](https://github.com/orgs/camunda/projects/57).
+
+The `camunda/issues` repository has two purposes:
+
+1. **Communicating our [product roadmap](https://github.com/orgs/camunda/projects/57)**: this is where you can learn about what features we're working on, what stage they're in, and when we expect to bring them to you. Have any questions or comments about items on the roadmap? Share your feedback by leaving a comment on the issue you are interested in.
+
+2. **Giving you a voice**: If you have a bug to report or a suggestion to propose, then (a) visit the respective [component repository](https://github.com/camunda/issues#public-product-components) listed below and open an issue there or (b) if there is no public repository, [open the issue here](https://github.com/camunda/issues/issues/new/choose).
+
+## Guide to the roadmap
```

Comment:
Let's please make this consistent.
Currently you talk about `target:xyz` and also about `potential release version`

Reply:
Additionally, you could use deep link like following to let users easily discover the concrete labels you are talking about:

Example for potential: https://github.com/camunda/issues/labels?q=potential